doc/releases/v03.02.adoc

```diff
@@ -0,0 +1,42 @@
+= https://github.com/OpenSimulationInterface/open-simulation-interface/releases/tag/v3.2.0[v3.2.0 - OSI "Editorial Eaton"]
+
+https://www.youtube.com/watch?v=SJUhlRoBL8M[Think p(OSI)tive!]
+
+
```

Comment:
Fixed whitespace in context of PR #533 (9ac1d74d03234f5c351f4691c1accc995a9c33c8).